apm-hub-helm/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for apm-hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+deployment:
+  label: apm-hub
+
+image:
+  repository: docker.io/flanksource/apm-hub
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "apm-hub"
```

Comment:
I think we remove these name overrides - For any significant overrides required we w would recommend HelmRelease post-renderers

Reply:
Noted.

Reply:
Have only kept nameOverride: incase the name needs to be different from the .Chart.Name Have left a comment in the values file defining it's use as well.